features/support/env.rb

```diff
@@ -50,6 +50,7 @@ def debrand_string(str)
         when "Create Kickstart Profile" then str
         when "Test Erratum" then str
         # replacement exceptions
+        when "Bare Metal Kickstart" then "Unprovisioned Autoinstallation"
```

Comment:
we want to get ride of debrand string function , see on issues, for the historical idea. So we dont need the changes there, but directly on page